application-services/custom/camera-management/appcamera/commands.go

```diff
@@ -7,7 +7,9 @@ package appcamera
 
 import (
 	"context"
-	"encoding/json"
+	"fmt"
```

Comment:
Even this file needs the copyright change.

Reply:
good catch. i forgot to update them as i went, so i was planning on going back through the pr and finding them.